src/app/account/components/tabs/connected-accounts/index.tsx

```
@@ -1,77 +1,243 @@
+  // },
+];
+
+
+export const ConnectedAccounts = () => {
+  const [connectedAccounts, setConnectedAccounts] = useState<ComposioConnection[]>([]);
+  const [availableServices] = useState<ServiceDefinition[]>(INITIAL_AVAILABLE_SERVICES);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [error, setError] = useState<string | null>(null);
+
+  // For individual button loading states
+  const [isConnecting, setIsConnecting] = useState<{ [serviceId: string]: boolean }>({});
```

ESLint

[error] A record is preferred over an index signature.

(@typescript-eslint/consistent-indexed-object-style)